redhat-cloud-client-configuration.spec

```diff
@@ -213,6 +231,138 @@ fi
 %endif
 
 
+%post cdn
+# insights-client
+%systemd_post insights-register.path
+%systemd_post insights-unregister.path
+%systemd_post insights-unregistered.path
+#rhcd
```

Comment:
Extreme nitpick: missing space after the hash sign.

Reply:
That's the same as in the `%post` for rhccc; since rhccc-cdn was started as copy/paste from rhccc, then also this came with it.

Reply:
Right, but could we have it prettier here, if not there? :)